engine/message/db.go

```diff
@@ -277,6 +279,23 @@ func NewDB(ctx context.Context, db *sql.DB, c *Config, a alertlog.Store) (*DB, e
 			) select distinct msg_id, alert_id, user_id, cm_id from disabled
 		`),
 
+		twilioDisabled: p.P(`
+			with disabled as (
+				update outgoing_messages msg
+				set
+					last_status = 'failed',
+					last_status_at = now(),
+					status_details = 'twilio disabled',
+					cycle_id = null,
+					next_retry_at = null
+				from user_contact_methods cm
+				where
+					msg.last_status = 'pending' and
+					msg.message_type != 'verification_message'
```

Comment:
We need to join against the contact method table and assert the type, anything going to voice or SMS will fail if Twilio is not configured

```suggestion
				join user_contact_methods cm on cm.id = msg.contact_method_id
				where
					msg.last_status = 'pending' and
					cm.type in ('SMS', 'VOICE')
```